js/ui/components/clubhouse.js

```diff
@@ -628,16 +631,31 @@ var ClubhouseOpenButton = new Lang.Class({
 
     _init: function(params) {
         params = params || {};
-        this._icon = new St.Icon({ style_class: 'clubhouse-open-button-icon' });
-        params.child = this._icon;
+        let gfile =
+            Gio.File.new_for_uri('resource:///org/gnome/shell/theme/clubhouse-icon-pulse.png');
+        this._pulseAnimation = new Animation(gfile,
+                                             CLUBHOUSE_BUTTON_SIZE,
+                                             CLUBHOUSE_BUTTON_SIZE,
+                                             CLUBHOUSE_BUTTON_PULSE_SPEED);
+        this._pulseIcon = this._pulseAnimation.actor;
+
+        this._normalIcon = new St.Icon({ style_class: 'clubhouse-open-button-icon',
+                                         width: CLUBHOUSE_BUTTON_SIZE,
+                                         height: CLUBHOUSE_BUTTON_SIZE
```

Comment:
Can you not set `icon-size` in the theme instead of hardcoding this here?

Reply:
Good catch! I have updated the PR. Thanks.